interfaces/apparmor/backend_test.go

```diff
@@ -341,22 +341,29 @@ var combineSnippetsScenarios = []combineSnippetsScenario{{
 	// Classic confinement uses apparmor in complain mode by default.
 	opts:    interfaces.ConfinementOptions{Classic: true},
 	snippet: "snippet",
-	content: commonPrefix + "\nprofile \"snap.samba.smbd\" (attach_disconnected,complain) {\nsnippet\n}\n",
+	content: "\n(classic)" + commonPrefix + "\nprofile \"snap.samba.smbd\" (attach_disconnected,complain) {\nsnippet\n}\n",
 }, {
 	// Classic confinement in JailMode uses enforcing apparmor.
 	opts:    interfaces.ConfinementOptions{Classic: true, JailMode: true},
 	snippet: "snippet",
-	content: commonPrefix + "\nprofile \"snap.samba.smbd\" (attach_disconnected) {\nsnippet\n}\n",
+	content: "\n(classic)" + commonPrefix + "\nprofile \"snap.samba.smbd\" (attach_disconnected) {\nsnippet\n}\n",
 }}
 
 func (s *backendSuite) TestCombineSnippets(c *C) {
 	// NOTE: replace the real template with a shorter variant
-	restore := apparmor.MockTemplate([]byte("\n" +
+	restoreTemplate := apparmor.MockTemplate([]byte("\n" +
+		"###VAR###\n" +
+		"###PROFILEATTACH### (attach_disconnected) {\n" +
+		"###SNIPPETS###\n" +
+		"}\n"))
+	defer restoreTemplate()
+	restoreClassicTemplate := apparmor.MockClassicTemplate([]byte("\n" +
+		"(classic)\n" +
```

Comment:
'(classic)' here means that the resulting profile won't parse which was confusing to me while performing this review. Is that intentional? Perhaps use '# (classic)' instead?